packages/next/server/load-components.ts

```diff
@@ -57,7 +59,8 @@ export async function loadDefaultErrorComponents(distDir: string) {
 export async function loadComponents(
   distDir: string,
   pathname: string,
-  serverless: boolean
+  serverless: boolean,
+  isFlight?: boolean
```

Comment:
are `getStaticProps`, `getStaticPaths` and `getServerSideProps` suppose to be available in flight in the future?
would it make sense to skip over anything aside from `Component` & `reactFlightManifest` when isFlight=true?